daemon/src/taker_cfd.rs

```diff
@@ -27,16 +19,15 @@ use model::Role;
 use model::Usd;
 use sqlite_db;
 use time::OffsetDateTime;
-use xtra::Actor as _;
 use xtra_productivity::xtra_productivity;
-use xtras::AddressMap;
+use xtras::SendAsyncSafe;
 
 #[derive(Clone)]
 pub struct CurrentMakerOffers(pub Option<MakerOffers>);
 
 #[derive(Clone, Copy)]
-pub struct TakeOffer {
-    pub order_id: OrderId,
+pub struct PlaceOrder {
+    pub offer_id: OrderId,
```

Comment:
maybe at least add a TODO to rollout `OfferId` in daemon later, when we decide to drop legacy protocols?

Reply:
This should actually be `OfferId` here :)